modules/animationqt/src/animationeditordockwidgetqt.cpp

```diff
@@ -190,6 +192,35 @@ AnimationEditorDockWidgetQt::AnimationEditorDockWidgetQt(
     toolBar->setFloatable(false);
     toolBar->setMovable(false);
     mainWindow_->addToolBar(toolBar);
+
+    // Menu
+    auto menu = mainWindow_->menuBar();
+
+    auto fileMenuItem = menu->addMenu(tr("&File"));
+    // file menu entries
+    {
+        auto openAction = new QAction(QIcon(":/svgicons/open.svg"), tr("&Open Animation"), this);
+        openAction->setShortcut(QKeySequence::Open);
```

Comment:
not sure this is a good idea. With this it depends on where you clicked last when trying to open a file. That is either the animation or workspace.
I'd consider this to be very confusing.

Reply:
Agree, if we do something like this is it has to go through the main app menu similar to how the edit menu works